### ISSUE TITLE
how can I just use yolov5s.pt

### ISSUE THREAD

**Issue by DarkLeaves** [ultralytics/yolov5#13334]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I want to use yolov5s.pt to do some experiments.
but when I run my code:
`model = YOLO('yolov5s.pt')`

the ultralytics will  Replace 'model=pics/yolov5s.pt' with new 'model=pics/yolov5su.pt'.
how to cancel this replacement?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @DarkLeaves, thank you for reaching out about YOLOv5 🚀!

For using `yolov5s.pt`, please ensure you're following the correct setup and process. You can find guidance in our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) to help you get started with different tasks, including the use of YOLOv5 models.

If this is a 🐛 **Bug Report**, please provide a **minimum reproducible example** to help us debug it. This will allow our team to better understand the problem.

If this is a ❓ **Question** related to custom use, please share detailed information, ensuring you're using our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

## Requirements

Please ensure your environment meets these requirements:

- [**Python>=3.8.0**](https://www.python.org/) with necessary [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed, including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Start with:
  ```bash
  git clone https://github.com/ultralytics/yolov5  # clone
  cd yolov5
  pip install -r requirements.txt  # install
  ```

## Environments

Consider running YOLOv5 in one of our verified environments:

- **Notebooks**: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status

Check our current status with this badge:
<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

Rest assured, an Ultralytics engineer will assist you soon. Meanwhile, explore using our latest [YOLOv8](https://github.com/ultralytics/ultralytics) for cutting-edge results!

Thanks again for your patience and interest 🌟

**Comment by pderrenger** [comment]
@DarkLeaves to prevent the automatic replacement, ensure you are using the correct version of YOLOv5 that supports your intended usage. You can specify the exact model path without triggering updates by using the `torch.hub.load` method like this:

```python
import torch
model = torch.hub.load('ultralytics/yolov5', 'custom', 'yolov5s.pt')
```

This should allow you to use `yolov5s.pt` without any automatic changes.
